content/docs/user-guide/managing-external-data.md

```diff
@@ -33,13 +34,13 @@ Currently, the following types (protocols) of external outputs (and
 - HDFS
 - Local files and directories outside the <abbr>workspace</abbr>
 
-> Note that these are a subset of the remote storage types supported by
-> `dvc remote`.
+💡 Note that external outputs are never pushed or pulled from/to
```

Comment:
it's not about note literally. It's about text becoming a list of disconnected paragraphs. Question- why do we have this paragraph here, after the list of supported types? Can we make it organically part of the text above?

Reply:
Hmmm, I'd prefer not to mix the previous paragraph with the concept of "remote storage". That will add confusion. I think that's why these mentions of remote storage are after the bullet list. I could possibly mix them into a more narrative paragraph (instead of descriptive notes)... ⌛

Reply:
it's not about the previous paragraph literally. It's about making it part of the some text above. Probably the one about `cache` works good. It can explain why does it happen. And even note about hash collision (I hope it won't be relevant) should go probably before the list of supported remote types.

Reply:
Yes, you didn't even mention the previous paragraph. I was also trying to refer to all the text before the list. I think mentioning remote storage in most places above may be confusing but let me try ⌛

Reply:
OK I mostly managed to do this, while also improving the explanations a little, I think. But I did have to leave one block quote note under the list of remote types (in the ext deps doc). PTAL @shcheklein